src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,157 @@
 ---
-title: Micrometer metrics registry
-tags:
-  - Integrations
-  - Open source telemetry integrations
-  - Micrometer
+title: Forward Micrometer data to New Relic with OpenTelemetry
 metaDescription: New Relic offers an integration that sends your Micrometer telemetry data to your New Relic account.
 redirects:
-  - /docs/integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry
-  - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-micrometer-integration
+    - /docs/integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry
+    - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-micrometer-integration
 freshnessValidatedDate: never
 ---
 
-[Micrometer](https://micrometer.io/) provides a simple facade over the instrumentation clients for the most popular monitoring systems, allowing you to instrument your JVM-based application code without vendor lock-in.
+import opentelemetryMicrometerWithOtel from 'images/opentelemetry_screenshot-full_micrometer-with-otel.webp'
 
-New Relic's Micrometer metrics registry sends your Micrometer telemetry data to your New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+[Micrometer](https://micrometer.io/) provides a simple facade over instrumentation clients, letting you instrument your JVM-based application code without vendor lock-in. You can configure the [OpenTelemetry Micrometer Bridge](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/micrometer/micrometer-1.5/library) along with the OpenTelemetry SDK to forward Micrometer data to New Relic via OTLP.
```

Comment:
@akristen I would remove this marketing-y intro sentence and replace it with stuff related to how New Relic can monitor it/what data we monitor/why its beneficial) (since we can assume they know what micrometer is by definition/they don't need high-level concept references)